app/v2/services/CreateAmendHistoricFhlUkPropertyAnnualSubmissionService.scala

```diff
@@ -50,13 +50,16 @@ class CreateAmendHistoricFhlUkPropertyAnnualSubmissionService @Inject()(connecto
 
   private def desErrorMap: Map[String, MtdError] =
```

Comment:
Can we rename this from `desErrorMap` to `downstreamErrorMap` as we are changing it from DES to IFS

Reply:
Can we move the **downstreamErrorMap**  into a companion object and then import into both service and tests to avoid duplication of errorMappings in both code + tests : https://github.com/hmrc/property-business-api/blob/b3d5974e1602261d3e282933b6fe473fc2e2a82a/app/v2/services/RetrieveHistoricNonFhlUkPropertyPeriodSummaryService.scala#L54-L67